node/node.go

```diff
@@ -681,6 +681,7 @@ func (n *Node) StartHeartbeat(config config.HeartbeatConfig) error {
 		n.privKey,
 		n.marshalizer,
 		HeartbeatTopic,
+		n.shardCoordinator.SelfId(),
```

Comment:
not used properly. Should have sent n.shardCoordinator.

Reply:
done